http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -687,4 +688,29 @@ class ClusterMonitorActor(
       _ <- metrics.incrementCounter(counterName)
       _ <- metrics.recordResponseTime(timerName, duration)
     } yield ()
+
+  private def findToolImageType(images: Set[ClusterImage]): String = {
+    val terraJupyterImage = "us.gcr.io/broad-dsp-gcr-public/([a-z0-9-_]+):(.*)".r
+    val anvilRStudioImage = "us.gcr.io/anvil-gcr-public/([a-z0-9-_]+):(.*)".r
+    images.find(clusterImage => Set(ClusterImageType.Jupyter, ClusterImageType.RStudio) contains clusterImage.imageType) match {
+      case Some(toolImage) => toolImage.imageUrl match {
+        case terraJupyterImage(imageType, _) => imageType
+        case anvilRStudioImage(imageType, _) => imageType
```

Comment:
I think we do care about `version` here as well so that we know whether a version bump degraded cluster creation time

Reply:
How would we represent that in new relic? Like having the version number be on the x-axis and the amount of time to create on y-axis?

Reply:
I think you can do `case terraJupyterImage(imageType, version) => "$imageType/$version"`? That'll give you something like `leonardo/ClusterMonitor/ClusterCreation/leonardo-jupyter/0.0.1/count`